src/geojson/geojson_intf.ml

```diff
@@ -215,6 +218,8 @@ module type S = sig
       (** [v features] creates a feature collection from a list of features *)
 
       include Json_conv with type t := t and type json := json
+
+      val foreign_members : t -> json
```

Comment:
We don't need individual `foreign_members` functions exposed to the users, the toplevel one is sufficient I think.